### PR TITLE
adding auth features to ui

### PR DESCRIPTION
**Pull request by Anand-Issac**

### CODE REVIEW

File: src/pages/User/Register.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import React, { useCallback, useState } from 'react';
+//import { useMutation, gql, useQuery } from '@apollo/client'
```
Comment (michaelmeng998) [inline src/pages/User/Register.js]: [nitpick]: can we remove the commented import

**Review by michaelmeng998 — COMMENTED** [review]
looking good! I made a nitpick. Also, the CI checks seems to be failing, can you look into that?

**Comment by Anand-Issac** [comment]
> looking good! I made a nitpick. Also, the CI checks seems to be failing, can you look into that?

Yes my mistake I had some warnings I did not check first (unused imports). I fixed it in my latest commit

**Review by michaelmeng998 — APPROVED** [review]
lgtm! nice work!

**Pull request merged**